### PR TITLE
Change the most spammy logs to `debug` and add progress bars in place.

### PR DESCRIPTION
**Pull request by garrettwrong**
also cleans up one or two comments.

I should have done this a long time ago :).

### CODE REVIEW

File: src/aspire/classification/rir_class2d.py [hunk 1]
```diff
@@ -334,7 +339,9 @@ def _sk_nn_classification(self, coeff_b, coeff_b_r):
 
     def _legacy_nn_classification(self, coeff_b, coeff_b_r):
         """
-        Perform nearest neighbor classification.
+        Perform nearest neighbor classification using port of ASPIRE legacy MATLAB code.
+
+        Note `distances` are dot products, ie "corr".
```
Comment (chris-langfield) [inline src/aspire/classification/rir_class2d.py]: regular quotes instead of `code quotes`?

**Review by chris-langfield — DISMISSED** [review]
Looks good just had a really pedantic question about `code quotes` vs "regular quotes" for a word that didn't seem to be referring to a variable

**Comment by garrettwrong** [comment]
@chris-langfield 

The distances are variable name for a variable in the returned tuple.  Do you think it should still be quotes?

**Comment by chris-langfield** [comment]
@garrettwrong Oh you're right sorry I didn't see that

**Comment by garrettwrong** [comment]
No worries. I updated the comment a little. Thanks!

File: src/aspire/classification/rir_class2d.py (hunk 1 above)
Comment (garrettwrong) [reply src/aspire/classification/rir_class2d.py]: Mentioned that its a returned var.

**Review by chris-langfield — APPROVED** [review]

**Pull request merged**